components/kubeflow/dnntrainer/build_image.sh

```diff
@@ -42,12 +50,18 @@ if [ -z "${TAG_NAME}" ]; then
   TAG_NAME=$(date +v%Y%m%d)-$(git describe --tags --always --dirty)-$(git diff | shasum -a256 | cut -c -6)
 fi
 
+TF_VERSION=1.6.0
```

Comment:
TF images have version, CPU/GPU and python version. Maybe we should just add the base image tag command line parameter.

Reply:
done